src/avtt/content-script.js

```diff
@@ -0,0 +1,580 @@
+        ["Components", request.components],
+        ["Range", request.range],
+        ["Casting Time", request["casting-time"]],
+        ["Duration", request.duration],
+        ...(request.ritual ? [["Ritual", "Yes"]] : []),
+        ...(request.concentration ? [["Concentration", "Yes"]] : []),
+    ];
+
+    let description = request.description;
+    let higher = description.indexOf("At Higher Levels.");
+    let higherDesc = null;
+    if (higher > 0) {
+        higherDesc = parseDescription(request, description.slice(higher + "At Higher Levels. ".length));
+        description = parseDescription(request, description.slice(0, higher - 1), {bulletList: false});
+    }
+
+    return {
+        icon: "torch",
+        color: "#BD10E0",
+        title: getName(request),
+        message: `${template(rolls)}
+
+${description}
+
+${(higher > 0 ? "**At Higher Levels.** " + higherDesc : "" )}
+` 
+    }
+}
+
+function rollSpellAttack(request, custom_roll_dice) {
+    let rolls = [
+        ["Level", request["cast-at"] !== undefined ? `${request["level-school"]} (Cast at ${request["cast-at"]} Level)` :  request["level-school"]],
+        ["Components", request.components],
+        ["Range", request.range],
+        ["Casting Time", request["casting-time"]],
+        ["Duration", request.duration],
+        ...(request.ritual ? [["Ritual", "Yes"]] : []),
+        ...(request.concentration ? [["Concentration", "Yes"]] : []),
+    ];
+
+    if (request["save-dc"] !== undefined) {
+        rolls = [
+            ...rolls,
+            ["Save DC", `DC ${request["save-dc"]} ${request["save-ability"]}`],
+        ]
+    }
+      
+    if (request["to-hit"] !== undefined) {
+       rolls = [
+            ...rolls,
+            ...advantageRoll(request, "To Hit", `!(1d20cr>=${request["critical-limit"] || 20}${request["to-hit"]}${custom_roll_dice.length > 0 ? " + " + custom_roll_dice : ""})`),
+        ]
+    }
+
+    if (request.damages !== undefined) {
+        const damages = request.damages;
+        const damage_types = request["damage-types"];
+        const critical_damages = request["critical-damages"] || [];
+        const critical_damage_types = request["critical-damage-types"] || [];
+        if (request.name === "Chromatic Orb") {
+            let chromatic_damage = null;
+            let crit_damage = null;
+            for (let dmgtype of ["Acid", "Cold", "Fire", "Lightning", "Poison", "Thunder"]) {
+                let idx = damage_types.findIndex(t => t === dmgtype);
+                chromatic_damage = damages.splice(idx, 1)[0];
+                damage_types.splice(idx, 1);
+                idx = critical_damage_types.findIndex(t => t === dmgtype);
+                if (idx >= 0) {
+                    crit_damage = critical_damages.splice(idx, 1)[0];
+                    critical_damage_types.splice(idx, 1)[0];
+                }
+            }
+            damages.splice(0, 0, chromatic_damage + "m2");
```

Comment:
+ "m2" ? Why are you adding that flag to the damage dice ? I couldn't see what it did.
Do you have a link to where I can see what the roll syntax is ?

Reply:
Found this : https://help.astraltabletop.com/dice/syntax/flags/
Looks like it's "match group" so it groups similar dice results together when you roll multiple dice.
In this case, I don't see why it's needed for the chromatic orb spell damage

Reply:
It should technically light up the damage if the dice match for an easier indication if the attack bounces again. Looks like I put it on chromatic org instead in chaos bolt. Either way tested it a bit and no highlighting appears, so I'll remove it.